### DISCUSSION TITLE
how does one power on crouton after installation

### DISCUSSION THREAD

**Discussion by bradfc** [dnschneid/crouton#4634] [Q&A] [answered]
I'm running a new Lenovo Flex 5 with Intel i3-1115G4 and just installed Ubuntu 16 + Unity.  Maybe this was already a mistake, but when I powered off and back on, Chrome put me in "recovery mode."  I found no way to launch an Ubuntu session.  It just treated me like a new user.  I hope that I have not somehow deleted an entire Ubuntu installation, but I can find no documentation how to get to it.  All the documentation I can finds tells how to install Ubuntu and "launch it," not how to get back to it after a power cycle.

I figure I'm missing something.

**Comment by n8-mills** [comment]
So, after power cycle you're opening CROSH and launching Ubunto from CHROOT?

**Comment by bradfc** [comment, marked as answer]
I think I've figured out how to do this.  There's no bug - I just had to go over the documentation several times.

Thanks for responding.
